docs/src/_includes/layouts/doc.html

```diff
@@ -19,6 +19,22 @@
     {% set added_version = rule_versions.added[title] %}
     {% set removed_version = rule_versions.removed[title] %}
 
+    {% if handled_by_typescript %}
+        {% set handled_by_typescript_content %}
+            <h2 id="handled_by_typescript">Handled by TypeScript</h2>
+            <p>
+                It is safe to disable this rule when using TypeScript because TypeScript's compiler enforces this check.
+            </p>
+            {% if extra_typescript_info %}
+                <p>
+                    {{ extra_typescript_info }}
```

Comment:
This will not interpret the content as Markdown, which means we will see the backticks. I believe this is the correct filter to use.
```suggestion
                    {{ extra_typescript_info | markdown }}
```